pkg/driver/node.go

```diff
@@ -254,14 +254,43 @@ func (d *nodeService) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 	if len(volumeID) == 0 {
 		return nil, status.Error(codes.InvalidArgument, "Volume ID not provided")
 	}
+	volumePath := req.GetVolumePath()
+	if len(volumePath) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "volume path must be provided")
```

Comment:
So prior to this PR, we don't have this requirement (at least here). We have to make sure this is not a breaking change. Did we verify that behavior stays the same when volumePath is absent?

Reply:
volume_path is required, so we should be fine I think.

Reply:
Yeah, as per https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/vendor/github.com/container-storage-interface/spec/lib/go/csi/csi.pb.go#L4322
I think we should have this check